### PR TITLE
[schema] use google struct

### PR DESCRIPTION
**Pull request by oded-soffrin**
Initial try

from Asana:
https://app.asana.com/0/1110053346854169/1202011419258561/f
The copied google struct.proto that was added back [then](https://github.com/wix/ricos/pull/3054/files#diff-3c137e29ebb5ba7806910a15cdf99510afa161d84c33a52b95017c1aa0aab266) breaks velo team code. The reason to copy google’s code was [this ts-proto issue](https://github.com/stephenh/ts-proto/issues/336). It has been fixed and is supposed to work now. We have to check how our types are generated by using newer ts-proto and importing google’s struct

### CODE REVIEW

File: packages/ricos-schema/web/tsconfig.json [hunk 1]
```diff
@@ -10,9 +10,15 @@
     "lib": ["dom", "ESNext"],
     "strictNullChecks": true,
     "esModuleInterop": true,
+    "downlevelIteration": true,
```
Comment (maimonav) [inline packages/ricos-schema/web/tsconfig.json:13]: Why is this needed?

File: packages/ricos-schema/web/package.json [hunk 2]
```diff
@@ -25,9 +25,9 @@
   "scripts": {
     "lint": "eslint 'src/**/*.{js,jsx,ts,tsx,d.ts}'",
     "clean": "rm -rf dist generated",
-    "prebuild": "yarn clean",
-    "generate": "ts-node --compiler-options={\\\"module\\\":\\\"commonJS\\\"} src/generateSchema.ts",
-    "gen-plugin-defaults": "ts-node --compiler-options={\\\"module\\\":\\\"commonJS\\\"} src/generate-plugin-defaults.ts",
+    "prebuild": "yarn clean",    
+    "generate": "ts-node src/generateSchema.ts",
```
Comment (maimonav) [inline packages/ricos-schema/web/package.json:29]: Like 🔥

**Comment by maimonav** [comment]
@oded-soffrin @lxgreen Do we have a test case for External schema to make sure that it's working properly?

File: packages/ricos-schema/web/tsconfig.json (hunk 1 above)
Comment (oded-soffrin) [reply packages/ricos-schema/web/tsconfig.json:13]: There was an error regarding proto wrapper code

```
Error:(125, 18) TS2569: Type 'Uint8Array' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.
```

**Comment by oded-soffrin** [comment]
> @oded-soffrin @lxgreen Do we have a test case for External schema to make sure that it's working properly?

Good question. Do we have a test case from schema changes all together ?

**Comment by oded-soffrin** [comment]
I thought about releasing a beta, but I guess it's not needed because a matching PR will be created in wix/private, right ?

**Comment by lxgreen** [comment]
well, we have converter tests that take care of external nodes - [toDraft](https://github.com/wix/ricos/blob/b9667f12e1e08c7b5437d195ca0a6f77e2c5c98a/packages/ricos-content/web/src/converters/draft/toDraft/toDraft.spec.ts#L243) and [fromDraft](https://github.com/wix/ricos/blob/b9667f12e1e08c7b5437d195ca0a6f77e2c5c98a/packages/ricos-content/web/src/converters/draft/fromDraft/__tests__/fromDraft.spec.ts#L353).

**Review by lxgreen — APPROVED** [review]
cool! you did order in balagan

**Review by maimonav — COMMENTED** [review]
Looks great!
So if we are covered by tests, please make sure you follow the [schema changes flow](https://bo.wix.com/pages/wix-ricos/docs/for-developers/schema-changes).

**Comment by oded-soffrin** [comment]
Still failing due to circular dependency
```
verb ricos-content build |  (!) Circular dependencies
verb ricos-content build |  ../../../node_modules/protobufjs/src/util/minimal.js -> ../../../node_modules/protobufjs/src/util/longbits.js -> ../../../node_modules/protobufjs/src/util/minimal.js
verb ricos-content build |  ../../../node_modules/protobufjs/src/util/minimal.js -> ../../../node_modules/protobufjs/src/util/longbits.js -> /Users/odeds/Code/ricos/node_modules/protobufjs/src/util/minimal.js?commonjs-proxy -> ../../../node_modules/protobufjs/src/util/minimal.js
```

**Comment by oded-soffrin** [comment]
> Still failing due to circular dependency
> 
> ```
> verb ricos-content build |  (!) Circular dependencies
> verb ricos-content build |  ../../../node_modules/protobufjs/src/util/minimal.js -> ../../../node_modules/protobufjs/src/util/longbits.js -> ../../../node_modules/protobufjs/src/util/minimal.js
> verb ricos-content build |  ../../../node_modules/protobufjs/src/util/minimal.js -> ../../../node_modules/protobufjs/src/util/longbits.js -> /Users/odeds/Code/ricos/node_modules/protobufjs/src/util/minimal.js?commonjs-proxy -> ../../../node_modules/protobufjs/src/util/minimal.js
> ```

Resolved thanks to release made by ts-proto 4 days ago 🍀
